### ISSUE TITLE
Document native apps in place of 'Quickstart Terminal'

### ISSUE THREAD

**Issue by cboettig** [ropenscilabs/r-docker-tutorial#40]
The native apps Docker now provides for Mac and Windows avoid the whole need for `Quickstart Terminal` and the cumbersome `docker-machine` calls to get the ip.  The whole thing works basically as it does on linux, just have the docker service running in the background and you can access docker through any terminal.  The RStudio session is available directly from `localhost:8787` (or whatever port you bind), no need to look up `ip`s.

**Comment by pgensler** [comment]
Yes, this needs to be updated. With Docker4Mac(not sure of the version), there is no need to map ```docker-machine ip:<port>``` to connect, or whatever IP was given in Docker Quickstart Terminal. Instead, you only need ```localhost``` to connect.
https://store.docker.com/editions/community/docker-ce-desktop-mac
